### DISCUSSION TITLE
Align History graphs

### DISCUSSION THREAD

**Discussion by jlpouffier** [home-assistant/frontend#13355] [Other feature requests] [answered]
When displaying multiple entities in the history tab, the multiple  x-axis are not aligned, which can make some analysis difficult and misleading.

Have a look at this picture and look at the 2 data point at 17:00 ...

![image](https://user-images.githubusercontent.com/5878296/183304241-abc1b451-85e1-444d-b7db-8816bf9997fd.png)

Confusing, right ?

_FYI : I decided to open a discussion and not a bug, because I don't think it's a bug per se... 
Nothing is broken, it's just confusing_

**Comment by ildar170975** [comment, marked as answer]
https://github.com/home-assistant/frontend/issues/3707
https://github.com/home-assistant/frontend/issues/11011
https://github.com/home-assistant/frontend/issues/10234
https://github.com/home-assistant/frontend/issues/10508
https://github.com/home-assistant/frontend/issues/13176
https://github.com/home-assistant/frontend/discussions/10296
https://github.com/home-assistant/frontend/discussions/11265
https://community.home-assistant.io/t/request-common-or-aligned-time-axis-on-all-history-graphs/246613

**Reply by zsarnett** [reply]
https://github.com/home-assistant/frontend/discussions/11265

Main Discussion should be held here
